ops/op2/dispatch_slow.rs

```diff
@@ -198,13 +203,31 @@ pub fn from_arg(
       }
     }
     Arg::Option(Special::String) => {
+      *needs_scope = true;
+      quote! {
+        let #arg_ident = #arg_ident.to_rust_string_lossy(#scope);
+      }
+    }
+    Arg::Special(Special::String) => {
+      *needs_scope = true;
       quote! {
-        let #arg_ident = #arg_ident.to_rust_string_lossy();
+        let #arg_ident = #arg_ident.to_rust_string_lossy(#scope);
       }
     }
     Arg::Special(Special::RefStr) => {
+      *needs_scope = true;
+      quote! {
+        // Trade 1024 bytes of stack space for potentially non-allocating strings
+        let mut #arg_temp: [::std::mem::MaybeUninit<u8>; 1024] = [::std::mem::MaybeUninit::uninit(); 1024];
```

Comment:
Should we keep it at 1024 or maybe go with something bigger like 8Kb?

Reply:
Let's leave it at 1024 for now (which should cover a pretty big chunk basic strings) and then later on we can experiment with the right size with some test cases to figure out the right balance.